pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -745,6 +747,9 @@ func TestReconcile(t *testing.T) {
 			if vapBindingCreationTime.Before(blockTime) {
 				return fmt.Errorf("VAPBinding should be created after default wait")
 			}
+			if ct.GetAnnotations()[constraint.VAPBGenerationAnnotation] == constraint.VAPBGenerationUnblocked {
+				return fmt.Errorf("expected %s annotations on CT to be unblocked", constraint.VAPBGenerationAnnotation)
```

Comment:
shouldnt the expected to be the opposite?

Reply:
Fixed test